packages/devtools_app/lib/src/debugger/debugger_model.dart

```diff
@@ -785,6 +785,13 @@ class Variable extends TreeNode<Variable> {
   }
 
   bool _isInspectable;
+
+  @override
+  Variable shallowCopy() {
+    throw UnimplementedError('This method is not implemented for an instance of'
```

Comment:
nit: Make these methods a bit more terse.No need to indicate which method was unimplemented as it is obvious from the call stack.